Postman tests/Media Watchlist API calls.json

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
I added a postman json file of my API calls so that you don't have to figure out how to do it. You may need to populate certain Environment variables (such as user id to create the Watchlist).